src/lab12.py

```diff
@@ -108,14 +108,18 @@ def __init__(self, tab):
             self.setTimeout)
         self.interp.export_function("requestAnimationFrame",
             self.requestAnimationFrame)
+        self.tab.browser.measure.time('script-runtime')
         self.interp.evaljs(RUNTIME_JS)
+        self.tab.browser.measure.stop('script-runtime')
 
         self.node_to_handle = {}
         self.handle_to_node = {}
 
     def run(self, script, code):
         try:
+            self.tab.browser.measure.time('script-load')
```

Comment:
I say we don't do this, but in principle we could extend `time` to also take an argument, and then these could be `script` blocks with different arguments. I think the chrome://tracing tools then let you aggregate all `script`s and look at the various arguments. Anyway, I don't think we should actually do it.